client/package.json

```diff
@@ -246,6 +245,7 @@
 		"@testing-library/jest-dom": "^5.14.1",
 		"@testing-library/react": "^12.0.0",
 		"@testing-library/react-hooks": "7.0.1",
+		"@types/jest": "^26.0.24",
```

Comment:
Does this need to match `jest` version?

Reply:
It was previously on v25 (with just at v27), so it wasn't causing issues. I had to set it to 26 because `emotion-jest` sets `@types/jest@26` as a peer dependency.

Reply:
is there a better way to handle that?

Reply:
Not really, if the authors of `@emotion/jest` really want v26, I'd say we go with that unless we have a very strong reason to override their decision.

That being said, I don't understand why they would want a specific version for _types_, as we won't be type checking `@emotion/jest` but our own code 🤷🏼

Reply:
Yeah, I have no idea: https://github.com/emotion-js/emotion/blob/main/packages/jest/package.json#L27